websites/L/Leosight/presence.ts

```diff
@@ -0,0 +1,456 @@
+      presenceData.startTimestamp = time;
+      if (path.includes("prijate")) presenceData.state = "Přijaté";
+      else if (path.includes("napsat")) presenceData.state = "Napsat e-mail";
+      else if (path.includes("odeslane")) presenceData.state = "Odeslané";
+      else if (path.includes("nastaveni")) presenceData.state = "Nastavení";
+  }
+  ////
+  else if (path.includes("tcu.ic")) {
+      presenceData.state = "Tax Compliance Unit";
+      presenceData.startTimestamp = time;
+  } else if (path.includes("mining.ic")) {
+      presenceData.state = "Těžba kryptoměn";
+      presenceData.startTimestamp = time;
+  } else if (path.includes("vaos.ic")) {
+      presenceData.state = "Veterinární a odchytová služba";
+      presenceData.startTimestamp = time;
+  } else if (path.includes("writer.ic")) {
+      presenceData.state = "Writer";
+      presenceData.startTimestamp = time;
+  }
+}
+//ctf//
+else if (window.location.href.includes("ctf")) {
+  presenceData.smallImageKey = "ctf";
+  presenceData.smallImageText = "ctf.leosight.cz";
+  presenceData.details = "Leosight CTF";
+  presenceData.startTimestamp = time;
+  if (path.includes("login.php")) presenceData.state = "Přihlášení";
+  else if (path.includes("index.php"))
+      presenceData.state = "Odvařuje si mozek";
+}
+//eco//
+else if (window.location.href.includes("eco")) {
+  presenceData.smallImageKey = "eco";
+  presenceData.smallImageText = "eco.leosight.cz";
+  presenceData.details = "Hraje Leosight ECO";
+  presenceData.startTimestamp = time;
+}
+//guard//
+else if (window.location.href.includes("guard")) {
+  presenceData.smallImageKey = "guard";
+  presenceData.smallImageText = "guard.leosight.cz";
+  presenceData.details = "Leosigh Guard";
+  presenceData.startTimestamp = time;
+  if (path === "" || path === "/") presenceData.state = "Seznam serverů";
+  if (path.includes("hive.php")) presenceData.state = "LSG - Hive";
+  else if (path.includes("skimose")) presenceData.state = "Skimo SE - API";
+  else if (path.includes("dokumentace.php"))
+      presenceData.state = "Skimo SE - Public API";
+  else if (path.includes("servery.php"))
+      presenceData.state = "LSEC ServerList";
+  else if (path.includes("license.php"))
+      presenceData.state = "LSG LicesnseList";
+  else if (path.includes("breach.php")) presenceData.state = "LSG-Breach";
+  else if (path.includes("prospect.php"))
+      presenceData.state = "Leosight Prospect";
+  else if (path.includes("/prospect/")) {
+      presenceData.details = "LeoSight rádio";
+      presenceData.startTimestamp = time;
+      if (path.includes("radio_mobile.php"))
+          presenceData.state = "Poslouchá rádio";
+      else if (path.includes("radio_newsong.php"))
+          presenceData.state = "Přidává novou písničku";
+  }
+}
+
+//neco//
+if (presenceData.details === null) {
+  presence.setTrayTitle("Provádí nespecifikovanou činnost.");
+  presence.setActivity();
+} else 
+  presence.setActivity(presenceData);
+
```

Comment:
```suggestion
  presence.setActivity(presenceData);
```

Reply:
Well every time I use beautifier, it comes with more curly problems

Reply:
No, that's not removing the curlies, it is just refactored code

Reply:
Well, idk what to do, I have formatted the text for X times and there's still the problem (Closing curly brace does not appear on the same line as the subsequent block. (brace-style)). I'll better close the request, I'm getting bullied by curly braces.